libs/features/personalization/personalization.js

```diff
@@ -655,17 +655,19 @@ function parsePlaceholders(placeholders, config, selectedVariantName = '') {
     'value',
     'other',
   ];
-  const [val] = Object.entries(placeholders[0])
```

Comment:
it was breaking here, where it was trying to deconstruct an array that was found by `find` function. But nothing was found `const [val]` would break because undefined is treated like an array